gateway/rest/types.go

```diff
@@ -5,10 +5,10 @@ import (
 )
 
 type LeasedIPStatus struct {
-	Port         uint32
-	ExternalPort uint32
-	Protocol     string
-	IP           string
+	Port         uint32 `json:"port"`
+	ExternalPort uint32 `json:"external_port"`
```

Comment:
this is potentially breaking change as the default values are different
https://go.dev/play/p/nh3PaU5Qatb